firmware/application/apps/ui_mictx.cpp

```diff
@@ -607,6 +607,36 @@ MicTXView::MicTXView(
     audio::input::start(ak4951_alc_and_wm8731_boost_GUI);  // When detected AK4951 => set up ALC mode; when detected WM8731 => set up mic_boost ON/OFF.
 }
 
+MicTXView::MicTXView(
```

Comment:
Here's the actual new code change. This is the first pass at supporting setting modulation and bandwidth overrides for MicTX (so you can jump from Recon/Scanner into Mic with the correct frequency and modulation)